app/models/status.rb

```diff
@@ -26,6 +24,8 @@
 #
 
 class Status < ApplicationRecord
+  self.cache_versioning = false
```

Comment:
It may be better to stop changing cache_versioning and stop using cache_key directly.